packages/dd-trace/src/log.js

```diff
@@ -11,6 +11,38 @@ let _logger
 let _enabled
 let _deprecate
 
+/*
+default [error, debug]
+from: https://tools.ietf.org/html/rfc5424
+
+  0       Emergency: system is unusable
+  1       Alert: action must be taken immediately
+  2       Critical: critical conditions
+  3       Error: error conditions
+  4       Warning: warning conditions
+  5       Notice: normal but significant condition
+  6       Informational: informational messages
+  7       Debug: debug-level messages
+*/
+
+let _customLogLevels
+
+const _isLogLevelEnabled = (level) => { return !_customLogLevels || _customLogLevels.indexOf(level) >= 0 }
+
+const _setCustomLogLevels = (customLogLevels) => {
+  if (customLogLevels) {
+    try {
+      if (typeof customLogLevels === 'string') {
+        return customLogLevels.toLowerCase().split(',')
+      } else if (Array.isArray(customLogLevels)) {
+        return customLogLevels.map(level => level.toLowerCase())
+      }
+    } catch (e) {
+      console.warn('customlogLevels option malformed', customLogLevels, e) /* eslint-disable-line no-console */
```

Comment:
The logger should always be used to log, even from within the logger.

Reply:
removed this log line after updating to `logLevel` as it's no longer necessary